### PR TITLE
[PM-2066] Remove waste logging

### PR DESCRIPTION
**Pull request by KonradStaniec**

### CODE REVIEW

**Review by pslaski — APPROVED** [review]

File: scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupInternals.scala [hunk 1]
```diff
@@ -52,18 +52,16 @@ private[dynamictls] object DynamicTLSPeerGroupInternals {
...
           case Left(value) =>
-            log.debug("Unexpected decoding error {} from peer {}", value: Any, ctx.channel().remoteAddress(): Any)
+            log.info("Unexpected decoding error {} from peer {}", value: Any, ctx.channel().remoteAddress(): Any)
             messageSubject.onNext(DecodingError)
 
           case Right(value) =>
-            log.debug("Decoded {} messages from peer {}", value.size, ctx.channel().remoteAddress())
-            value.foreach(m => messageSubject.onNext(MessageReceived(m)))
+            value.foreach { m =>
+              log.info("Decoded new message from peer {}", ctx.channel().remoteAddress())
```
Comment (ntallar) [inline scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupInternals.scala]: Shouldn't this be on debug?

File: scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupInternals.scala [hunk 2]
```diff
@@ -52,18 +52,16 @@ private[dynamictls] object DynamicTLSPeerGroupInternals {
...
       val byteBuf = msg.asInstanceOf[ByteBuf]
       try {
-        log.info(
-          s"Processing inbound message from remote address ${ctx.channel().remoteAddress()} " +
-            s"to local address ${ctx.channel().localAddress()}"
-        )
         codec.streamDecode(BitVector(byteBuf.nioBuffer())) match {
           case Left(value) =>
-            log.debug("Unexpected decoding error {} from peer {}", value: Any, ctx.channel().remoteAddress(): Any)
+            log.info("Unexpected decoding error {} from peer {}", value: Any, ctx.channel().remoteAddress(): Any)
```
Comment (ntallar) [inline scalanet/src/io/iohk/scalanet/peergroup/dynamictls/DynamicTLSPeerGroupInternals.scala]: Should this be an error msg?

**Review by ntallar — CHANGES_REQUESTED** [review]
How much of an improvement is removing this logging? Maybe we should eventually remove more so as to continue improving it?

**Comment by KonradStaniec** [comment]
It is hard to quantify tbh, i am removing only this one as it was only one which popped out on profiler during sync, if there will any other after that then we can remove it as well

File: build.sc [hunk 3]
```diff
@@ -14,7 +13,7 @@ object scalanet extends ScalaModule with PublishModule {
 
   def scalaVersion = "2.12.10"
 
-  def publishVersion = "0.1.11-SNAPSHOT"
+  def publishVersion = "0.1.12-SNAPSHOT"
```
Comment (ntallar) [inline build.sc]: This should now be updated to 0.1.13-SNAPSHOT, right?

**Review by ntallar — CHANGES_REQUESTED** [review]

**Review by ntallar — APPROVED** [review]
LGTM!
```
     /(__M__)\
    /, ,   , ,\
   /' ' 'V' ' '\       ARV

------------------------------------------------
Thank you for visiting https://asciiart.website/
This ASCII pic can be found at
https://asciiart.website/index.php?art=animals/bats
```

**Pull request merged**